citations/stat.js

```diff
@@ -29,13 +29,12 @@ module.exports = {
   ],
 
   links: function(cite) {
-    // GPO only has volumes starting with 65.
-    if (cite.volume < 65) return { };
+    var links = { };
 
-    var usgpo_url = "http://api.fdsys.gov/link?collection=statute&volume=" + cite.volume + "&page=" + cite.page;
-
-    return {
-      usgpo: {
+    // GPO
+    if (cite.volume >= 65 && cite.volume <= 125) {
```

Comment:
In that case, my preference is to remove the ceiling, and put the burden on software that integrates with this to employ such a ceiling (and keep it up to date) in their own code. Do you (or anyone else) feel strongly otherwise?